landcover.mss

```diff
@@ -340,13 +341,18 @@
   [feature = 'natural_grassland'][zoom >= 8],
   [feature = 'landuse_meadow'][zoom >= 8],
   [feature = 'landuse_grass'][zoom >= 10],
+  [feature = 'landcover_grass']
```

Comment:
A comma `,` is missing at the end of the line.

Reply:
Fixed